### PR TITLE
SpeedyBee F405 Mini

### PR DESCRIPTION
**Pull request by andyp1per**
https://www.speedybee.com/speedybee-f405-mini-bls-35a-20x20-stack/

### CODE REVIEW

**Comment by tridge** [comment]
ping @Hwurzburg

**Review by tridge — APPROVED** [review]
looks good, but @Hwurzburg may wish to check docs

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (DJI-VTX, DMA-enabled)
+ - SERIAL2 -> UART2 (RX, DMA-enabled) 
+ - SERIAL3 -> UART3
+ - SERIAL4 -> UART4 (connected to internal BT module, not currently usable by ArduPilot)
+ - SERIAL5 -> UART5 (ESC Telemetry)
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:29]: should note RX pin only, on ESC connector

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+LiPo batteries.
+
+The default battery parameters are:
+
+ - BATT_MONITOR 4
+ - BATT_VOLT_PIN 10
+ - BATT_CURR_PIN 11
+ - BATT_VOLT_MULT 11.2
+ - BATT_AMP_PERVLT 40 (will need to be adjusted for whichever current sensor is attached)
+
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:83]: ## RSSI
Analog RSSI input (pin 15)

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
looks good...just a couple of nitpicks (primarily for me) so that I don't have to scan the wiring diagram and hwdef again when making the wiki page

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md (hunk 2 above)
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:83]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:29]: how?where?

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:83]: how?where?

**Comment by Hwurzburg** [comment]
@andyp1per did you forget to push the changes???? I unresolved my comments pending that

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md (hunk 1 above)
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md]: No idea - I redid them

**Comment by Hwurzburg** [comment]
RSSI one also....thanks

File: libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md (hunk 2 above)
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md:83]: Again

**Comment by Hwurzburg** [comment]
yeah...thanks

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**